src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/XmlElementElementCollection.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Configuration;
+using System.Xml;
+
+namespace CoreWCF.Configuration
+{
+    [ConfigurationCollection(typeof(XmlElementElement), AddItemName = ConfigurationStrings.XmlElement, CollectionType = ConfigurationElementCollectionType.BasicMap)]
+    public sealed class XmlElementElementCollection : ServiceModelConfigurationElementCollection<XmlElementElement>
+    {
+        public XmlElementElementCollection()
+            : base(ConfigurationElementCollectionType.BasicMap, ConfigurationStrings.XmlElement) { }
+
+        protected override object GetElementKey(ConfigurationElement element)
+        {
+            if (element == null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("element");
+            }
+
+            return ((XmlElementElement)element).XmlElement.OuterXml;
+        }
+
+        protected override void Unmerge(ConfigurationElement sourceElement,
+                                                 ConfigurationElement parentElement,
```

Comment:
I know it wasn't in the original code, but can you line up the parameters on the same text column?